pixl_core/tests/conftest.py

```diff
@@ -53,7 +53,7 @@ def run_containers() -> subprocess.CompletedProcess[bytes]:
         timeout=60,
     )
     subprocess.run(
-        b"docker compose down --volumes",
+        b"docker compose down",
```

Comment:
Ah intersting, why have we removed this?

Reply:
I had a (wrong) idea about fixing system tests.